### PR TITLE
Add message for november 12th

### PR DESCRIPTION
**Pull request by smessie**

### CODE REVIEW

File: server/scraper/resto/menu_manual.py [hunk 1]
```diff
@@ -328,6 +328,16 @@ def brug_avond(_path, original):
     original["message"] = "De Brug is vrijdagavond gesloten."
     return original
 
+def november_12_2021(_path, original):
+    original["message"] = "Vrijdag 12 november zullen alle resto's en cafetaria's gesloten zijn, behalve cafetaria De Brug."
+    original["open"] = False
+    return original
+
+def november_12_2021_en(_path, original):
+    original["message"] = "Vrijdag 12 november zullen alle resto's en cafetaria's gesloten zijn, behalve cafetaria De Brug."
```
Comment (niknetniko) [inline server/scraper/resto/menu_manual.py:337]: This message should be in English.

File: server/scraper/resto/menu_manual.py [hunk 2]
```diff
@@ -611,6 +621,20 @@ def create_changes(root_path):
             resto="nl-debrug",
             start=date(2021, 10, 1),
             end=date(2021, 12, 31)
+        ),
+        ManualChange(
+            replacer=november_12_2021,
+            resto=["nl", "nl-heymans", "nl-dunant", "nl-coupure", "nl-sterre", "nl-ardoyen", "nl-merelbeke"],
```
Comment (niknetniko) [inline server/scraper/resto/menu_manual.py:627]: There is only `nl` nowadays, so it should be enough to only add `nl`.

**Review by niknetniko — CHANGES_REQUESTED** [review]
Thanks a lot! It looks pretty good, except for two small things.

**Comment by smessie** [comment]
> Thanks a lot! It looks pretty good, except for two small things.

Should be fixed now :)

**Review by niknetniko — APPROVED** [review]

**Pull request merged**